### PR TITLE
Response returns long.

### PR DESCRIPTION
**Pull request by rafe-g**
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

**Comment by rafael-chavez** [comment]
@lukeis

File: java/client/src/org/openqa/selenium/DeviceRotation.java [hunk 1]
```diff
@@ -54,14 +54,14 @@ public DeviceRotation(int x, int y, int z) {
...
-	public DeviceRotation(Map<String, Integer> map) {
+	public DeviceRotation(Map<String, Long> map) {
 		if (map == null || !map.containsKey("x") || !map.containsKey("y") || !map.containsKey("z")) {
 			throw new IllegalArgumentException("Could not initialize DeviceRotation with map given: " + map.toString());
 		}
-		this.validateParameters(map.get("x"), map.get("y"), map.get("z"));
-		this.x = map.get("x");
-		this.y = map.get("y");
-		this.z = map.get("z");
+		this.validateParameters(map.get("x").intValue(), map.get("y").intValue(), map.get("z").intValue());
```
Comment (lukeis) [inline java/client/src/org/openqa/selenium/DeviceRotation.java]: put the validate line at the end and then you can do just:

    this.validateParameters(x, y, z);

also, in validateParameters can you adjust it to ensure the values are less than 360 too (as per spec)

File: java/client/src/org/openqa/selenium/DeviceRotation.java [hunk 2]
```diff
@@ -54,14 +54,14 @@ public DeviceRotation(int x, int y, int z) {
 	 * z : zVal
 	 * @param map
 	 */
-	public DeviceRotation(Map<String, Integer> map) {
+	public DeviceRotation(Map<String, Long> map) {
```
Comment (lukeis) [inline java/client/src/org/openqa/selenium/DeviceRotation.java]: use Number

**Comment by rafael-chavez** [comment]
@lukeis thanks for reviewing, updated!

**Pull request merged**